frappe/model/document.py

```diff
@@ -709,6 +709,7 @@ def _evaluate_alert(alert):
 			# value change is not applicable in insert
 			event_map['validate'] = 'Value Change'
 			event_map['before_change'] = 'Value Change'
+			event_map['before_update_after_submit'] = 'Value Change'
```

Comment:
@rohitwaghchaure `on_update_after_submit` instead of `before_update_after_submit `?